relayer/src/worker/retry_strategy.rs

```diff
@@ -1,12 +1,93 @@
-use crate::util::retry::{clamp_total, ConstantGrowth};
+use crate::util::retry::{clamp_total, clamp, ConstantGrowth};
 use std::time::Duration;
 
-const MAX_DELAY: Duration = Duration::from_millis(500);
-const DELAY_INCR: Duration = Duration::from_millis(100);
-const INITIAL_DELAY: Duration = Duration::from_millis(200);
-const MAX_RETRY_DURATION: Duration = Duration::from_secs(2);
 
+/// A basic worker retry strategy.
+///
+/// The delay is initially 200ms and grows
+/// by 100ms at each step. The delay is
+/// capped at 500ms.
+/// The overall amount of time spent retrying
+/// is capped to 2 seconds.
+/// See the `default_strategy` test below.
 pub fn worker_default_strategy() -> impl Iterator<Item = Duration> {
-    let strategy = ConstantGrowth::new(INITIAL_DELAY, DELAY_INCR);
-    clamp_total(strategy, MAX_DELAY, MAX_RETRY_DURATION)
+    let strategy = ConstantGrowth::new(Duration::from_millis(200), Duration::from_millis(100));
+    clamp_total(strategy, Duration::from_millis(500), Duration::from_secs(2))
 }
+
+
+/// A stubborn worker retry strategy.
+///
+/// Initial retry delay is hardcoded to 1s, and
+/// the delay grows by 1s at every step. The
+/// strategy delay is capped at 60 sec. The
+/// total number of retries is capped to 400.
+/// Therefore the overall amount of time spent
+/// retrying will be approx. 6 hours.
+///
+/// See the `stubbord_strategy` test below.
+pub fn worker_stubborn_strategy() -> impl Iterator<Item = Duration> {
+    let strategy = ConstantGrowth::new(Duration::from_secs(1), Duration::from_secs(1));
+    clamp(strategy, Duration::from_secs(60), 400)
+}
+
+
+#[cfg(test)]
+mod tests {
+    use std::time::Duration;
+
+    use crate::worker::retry_strategy::{worker_stubborn_strategy, worker_default_strategy};
+
+    #[test]
+    fn default_strategy() {
```

Comment:
This test is almost identical to this test
https://github.com/informalsystems/ibc-rs/blob/85446586daf6dba652271eef31dae29f35da86c2/relayer/src/util/retry.rs#L153

but I only realized after writing it.

I still found to keep it, for the purpose of understanding exactly the details of the strategy we use in practice.